### PR TITLE
Add dcos-autoscaler package

### PR DESCRIPTION
**Pull request by Ahmadposten**
Adding package dcos-autoscaler to universe.

This package provides service autoscaling based on cpu or memory. Every app autoscaling behavior is configurable using labels similar to how marathon load balancer works.

### CODE REVIEW

**Comment by Ahmadposten** [comment]
@jeorg84 can you please take a look at this?

**Comment by Ahmadposten** [comment]
@ryadav88

File: repo/packages/D/dcos-autoscaler/0/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-autoscaler",
+  "version": "1.0.0",
+  "maintainer": "ahmadhassan612@gmail.com",
+  "description": "Adds autoscaling to your application based on cpu and memory metrics",
+  "scm": "https://github.com/Ahmadposten/DC-OS-Marathon-autoscaling",
+  "framework": true,
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: `framework` should be false

File: repo/packages/D/dcos-autoscaler/0/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "tags": ["autoscaling", "mesosphere", "dcos"],
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: `mesosphere` tag is reserved for packages developed by Mesosphere. Please remove it.

File: repo/packages/D/dcos-autoscaler/0/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: Add `preInstallNotes` and add this line at the beginning of `preInstallNotes`: ```This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!```

File: repo/packages/D/dcos-autoscaler/0/package.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  "packagingVersion": "3.0",
+  "name": "dcos-autoscaler",
+  "version": "1.0.0",
+  "maintainer": "ahmadhassan612@gmail.com",
+  "description": "Adds autoscaling to your application based on cpu and memory metrics",
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: Add a detailed description of what the package does, what it be used for.

File: repo/packages/D/dcos-autoscaler/0/package.json (hunk 3 above)
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: Add a link to `dcos/examples` at the end of `preInstallNotes`

File: repo/packages/D/dcos-autoscaler/0/package.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  "scm": "https://github.com/Ahmadposten/DC-OS-Marathon-autoscaling",
+  "framework": true,
+  "tags": ["autoscaling", "mesosphere", "dcos"],
+  "postInstallNotes": "Thank you for installing dcos autoscaler, Consult github.com/Ahmadposten/DC-OS-Marathon-autoscaling for more more info on usage and tuning configurations"
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/package.json]: Add the `license` property.

File: repo/packages/D/dcos-autoscaler/0/resource.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://drive.google.com/open?id=0BxADtivbfBMNd0VBbThQNmtkcUE",
+    "icon-medium": "https://drive.google.com/open?id=0BxADtivbfBMNd0VBbThQNmtkcUE",
+    "icon-large": "https://drive.google.com/open?id=0BxADtivbfBMNd0VBbThQNmtkcUE"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "23b1cfe8e04a": "ahmadposten/dcos-autoscaler"
```
Comment (ryadav88) [inline repo/packages/D/dcos-autoscaler/0/resource.json]: use a tagged version of the image

**Review by ryadav88 — CHANGES_REQUESTED** [review]
Please add an example documentation by creating a PR against the https://github.com/dcos/examples repo. See other examples there for reference. This is also now a requirement for submitting packages to Universe.

File: repo/packages/D/dcos-autoscaler/0/package.json (hunk 3 above)
Comment (Ahmadposten) [inline repo/packages/D/dcos-autoscaler/0/package.json]: @ryadav88  done .. please review again
here is the pull request for the examples repo https://github.com/dcos/examples/pull/170

**Pull request closed without merging**